pkg/didcomm/protocol/didexchange/models.go

```diff
@@ -15,7 +15,7 @@ import (
 type OOBInvitation struct {
 	// ID of this invitation (for record-keeping purposes).
 	// TODO can we remove this?
-	ID string
+	ID string `json:"@id"`
```

Comment:
TODO can we remove this? - sure, it will be set by the messenger. If you need this ID before the messenger sets it you need to do this manually. e.g

```
if msg.ID() == "" {
    msg.SetID(uuid.New().String())
}
```

Reply:
`OOBInvitation` is not a DIDComm message. I don't expect it to be sent nor received via the messenger.

Reply:
To be clear: that TODO is about removing `ID` entirely, not just the json tag.